utils/cr_demo/demo_utils.py

```diff
@@ -0,0 +1,396 @@
+def format_storage(host_list):
+    """Call dmg storage format.
+
+    Args:
+        host_list (str): List of hosts to format.
+    """
+    format_cmd = ["dmg", "storage", "format", "--host-list=" + host_list]
+    run_command(command=format_cmd)
+
+
+def storage_query_usage(host_list):
+    """Call dmg storage query usage.
+
+    Args:
+        host_list (str): List of hosts to query.
+    """
+    storage_query_cmd = ["dmg", "storage", "query", "usage", "--host-list=" + host_list]
+    run_command(command=storage_query_cmd)
+
+
+# Pool-related methods
+def create_pool(pool_size, pool_label, ranks=None, nsvc=None):
+    """Call dmg pool create.
+
+    Args:
+        pool_size (str): Pool size.
+        pool_label (str): Pool label.
+        ranks (str): Ranks to create pool. Defaults to None.
+        nsvc (str): Number of service replicas. Defaults to None.
+    """
+    create_pool_cmd = ["dmg", "pool", "create", pool_label, "--size=" + pool_size]
+    if ranks:
+        create_pool_cmd.append("--ranks=" + ranks)
+    if nsvc:
+        create_pool_cmd.append("--nsvc=" + nsvc)
+    run_command(command=create_pool_cmd)
+
+
+def list_pool(verbose=False, json=False, no_query=False):
+    """Call dmg pool list.
+
+    Args:
+        verbose (bool): Whether to use --verbose. Defaults to False.
+        json (bool): Whether to use --json. If used, verbose value would be irrelevant.
+            Defaults to False.
+        no_query (bool): Whether to use --no-query. Defaults to False.
+
+    Returns:
+        str: If --json is used, return stdout. Otherwise None.
+
+    """
+    if json:
+        list_pool_cmd = ["dmg", "--json", "pool", "list"]
+    else:
+        list_pool_cmd = ["dmg", "pool", "list"]
+    if verbose:
+        list_pool_cmd.append("--verbose")
+    if no_query:
+        list_pool_cmd.append("--no-query")
+    command = " ".join(list_pool_cmd)
+    print(f"Command: {command}")
+
+    if json:
+        result = subprocess.run(
+            list_pool_cmd, stdout=subprocess.PIPE, universal_newlines=True, check=False)
+        return result.stdout
+
+    subprocess.run(list_pool_cmd, check=False)
+    return None
+
+
+def pool_get_prop(pool_label, properties):
+    """Call dmg pool get-prop <pool_label> <properties>
+
+    Args:
+        pool_label (str): Pool label.
+        properties (str): Properties to query. Separate them with comma if there are
+            multiple properties.
+    """
+    get_prop_cmd = ["dmg", "pool", "get-prop", pool_label, properties]
+    run_command(command=get_prop_cmd)
+
+
+def pool_query(pool_label):
+    """Call dmg pool query
+
+    Args:
+        pool_label (str): Pool label.
+    """
+    pool_query_cmd = ["dmg", "pool", "query", pool_label]
+    run_command(command=pool_query_cmd)
+
+
+def list_containers(pool_label):
+    """Call daos pool list-containers <pool_label>
+
+    Args:
+        pool_label (str): Pool label.
+    """
+    list_containers_cmd = ["daos", "pool", "label", pool_label]
```

Comment:
Typo? But probably means this isn't used. Maybe remove the function?
```suggestion
    list_containers_cmd = ["daos", "pool", "list", pool_label]
```

Reply:
Removed this method.